src/Stratis.Bitcoin.Features.SignalR/Broadcasters/WalletInfoBroadcaster.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Connection;
+using Stratis.Bitcoin.Features.SignalR.Events;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+using Stratis.Bitcoin.Signals;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.SignalR.Broadcasters
+{
+    public class WalletInfoBroadcaster : ClientBroadcasterBase
+    {
+        private readonly IWalletManager walletManager;
+        private readonly IConnectionManager connectionManager;
+        private readonly Network network;
+        private readonly ChainIndexer chainIndexer;
+        private ILogger logger;
+
+        public WalletInfoBroadcaster(
+            ILoggerFactory loggerFactory,
+            IWalletManager walletManager,
+            IConnectionManager connectionManager,
+            ISignals signals,
+            ChainIndexer chainIndexer,
+            INodeLifetime nodeLifetime,
+            EventsHub eventsHub)
+            : base(eventsHub, signals, nodeLifetime, loggerFactory)
+        {
+            this.walletManager = walletManager;
+            this.connectionManager = connectionManager;
+            this.chainIndexer = chainIndexer;
+        }
+
+        protected override IEnumerable<IClientEvent> GetMessages()
+        {
+            WalletGeneralInfoClientEvent clientEvent = null;
+            try
```

Comment:
Maybe we could move `try and catch` inside each iteration of the `foreach` loop, as it would be more beneficial to log which particular wallet failed to load.

Reply:
Done